keyboards/ergodox_ez/keymaps/mac_win_german_programmer/keymap.c

```diff
@@ -0,0 +1,357 @@
+#include "ergodox_ez.h"
+#include "debug.h"
+#include "action_layer.h"
+#include "version.h"
+
+
+#include "keymap_german.h"
+
+#include "keymap_nordic.h"
+
+
+
+enum custom_keycodes {
+  PLACEHOLDER = SAFE_RANGE, // can always be here
```

Comment:
You don't actually need `PLACEHOLDER`, and could use: 

```suggestion
  VRSN = SAFE_RANGE, // can always be here
```